.github/workflows/release.yml

```diff
@@ -49,10 +49,11 @@ jobs:
           repository-url: https://test.pypi.org/legacy/
 
   publish-pypi:
-    name: Upload release to PyPI
+    name: Upload release to PyPI - only if test release is successful!
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v20')
     needs:
       - build-distribution
```

Comment:
I guess we don't technically need both of these, since `test-pypi` already depends on `build-distribution`